BackendWebApp/src/test/java/uk/ac/ebi/biostd/webapp/application/security/service/MagicFolderUtilTest.java

```diff
@@ -56,5 +61,29 @@ public void createGroupMagicFolder() {
         File magicFolder = new File(parentFolder, "c-123-b40");
         assertThat(magicFolder).exists();
     }
+
+    @Test
+    public void getUserMagicFolderPath() {
+        Path userMagicFolderPath = testInstance.getUserMagicFolderPath(TEST_ID, TEST_SECRET);
+        assertThat(userMagicFolderPath).isEqualTo(getExpectedPath(MagicFolderUtil.USER_FOLDER_PREFIX));
+    }
+
+    @Test
+    public void getGroupMagicFolderPath() {
+        Path userMagicFolderPath = testInstance.getGroupMagicFolderPath(TEST_ID, TEST_SECRET);
+        assertThat(userMagicFolderPath).isEqualTo(getExpectedPath(MagicFolderUtil.GROUP_FOLDER_PREFIX));
+    }
+
+    @Test(expected = NoSuchFileException.class)
```

Comment:
not big argument in this one but I would recommend to use 

```
assertThatExceptionOfType(NoSuchFileException.class)
                .isThrownBy(() -> faultyTestInstance.createUserMagicFolder(TEST_ID, TEST_SECRET);)
```

reasons:
- single mechanism used when need to validate exception message or other exception attributes
- consistent with junit5 (when we migrate)   `assertThrows` statment

Reply:
Got it. Fixed